### DISCUSSION TITLE
youtube МТС

### DISCUSSION THREAD

**Discussion by slesru** [hufrea/byedpi#183] [General]
Доброе утро!

наземный МТС перестал показывать ютуб,
всё что смог придумать:

-s1 -At -d2 -At -f-1 -r1+s -An --disorder 3 --ttl 5 --tlsrec 3+s --auto=torst --timeout 3

так показывает, но очень долго не начинает, секунд 30 а то и больше крутится, прежде чем вилео начнётся.
Может есть идеи, что сделать?

Спасибо!

**Comment by slesru** [comment]
Сегодня работать вообще перестало.

**Reply by EMWD** [reply]
подтверждаю

**Comment by slesru** [comment]
упомянутый тут в issues вариант работает
-d1 -r25+s

**Reply by slesru** [reply]
Перестал работать :-(

**Reply by slesru** [reply]
-d1 -fake-2 -r25+s
вот так работает, пока :-)

**Reply by slesru** [reply]
Опять сломали.
Ну понятно чо, спасибо шахназарову, царь то не знает.

**Reply by slesru** [reply]
split cancel: pos=1-0, n=1901

вот так пишет, что значит?

**Reply by slesru** [reply]
-d1 -r25+s
работает

**Reply by slesru** [reply]
опять сломали ироды

**Reply by slesru** [reply]
кстати ещё и гугль плей забанили

**Comment by marchcat69** [comment]
-s2 -d3 работает

**Reply by slesru** [reply]
нет
